bindings/c/test/mako/mako.c

```diff
@@ -1612,6 +1614,7 @@ void usage() {
 	printf("%-24s %s\n", "    --flatbuffers", "Use flatbuffers");
 	printf("%-24s %s\n", "    --streaming", "Streaming mode: all (default), iterator, small, medium, large, serial");
 	printf("%-24s %s\n", "    --disable_ryw", "Disable snapshot read-your-writes");
+	printf("%-24s %s\n", "    --json=PATH", "Output stats to the specified json file");
```

Comment:
This is great, but a bit confusing.
When `--json` is specified, the console output is still not JSON.
Maybe we should have `--stats-file=PATH` and `--stats-format={JSON|ASCII}` or something?